ps_mbo.php

```diff
@@ -232,13 +242,13 @@ public function installTab(array $tabData)
     }
 
     /**
-     * Uninstall Module.
+     * Disable Module.
      *
      * @return bool
      */
-    public function uninstall()
```

Comment:
Why removing the uninstall method?

Reply:
Because its content has been moved to the `disable()` function.

Reply:
But if we uninstall, does it trigger the disable method?

Reply:
[Yep](https://github.com/PrestaShop/PrestaShop/blob/develop/classes/module/Module.php#L728)!